index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css">
+    
     <style>
```

Comment:
Good job so far 

However you really need to use a a different file for your Css and style 

to make your project readable and better to maintain